pkg/updater/resultstore/resultstore.go

```diff
@@ -70,6 +73,117 @@ type processedResult struct {
 	TargetResults   map[string][]*singleActionResult
 }
 
+type timedError struct {
+	millis int64
+	err    error
+}
+type invocationGroup struct {
+	group       string
+	invocations []processedResult
+	errs        map[string]timedError
+}
+
+const (
+	// PartOf is the configuration value key which links to the group invocation ID.
+	PartOf = "EXECUTED_AS_PART_OF_INVOCATION"
+	// Originally indicates the associated non-overview indication.
+	Originally = "Original_test_invocation"
+)
+
+// buildIdentifier may assume its arguments are non-nil.
+type buildIdentifier func(configpb.TestGroup, processedResult) string
+
+var (
+	zero = time.Unix(0, 0)
+)
+
+func timeRunDate(milliseconds int64) time.Time {
+	return zero.Add(time.Duration(milliseconds) * time.Millisecond)
+}
+
+func invocationGroupID(inv processedResult, primary configpb.TestGroup_PrimaryGrouping, fallback configpb.TestGroup_FallbackGrouping, fallbackKey string, identifyBuild string) string {
+	if primary == configpb.TestGroup_PRIMARY_GROUPING_BUILD {
+		return identifyBuild
+	}
+	// PRIMARY_GROUPING_NONE, consider fallback
+	id := inv.InvocationProto.GetId().String()
```

Comment:
let's remove this - imo we don't need this here... we can add this later if need be. let's focus on primary and fallback groupings. only